### ISSUE TITLE
 __INTLIFY_PROD_DEVTOOLS__ is not defined

### ISSUE THREAD

**Issue by RaphBensimon** [intlify/vue-i18n-next#567]
### Reporting a bug?

I got this error in production : core-base.esm-bundler.js:167 Uncaught ReferenceError: __INTLIFY_PROD_DEVTOOLS__ is not defined

### Expected behavior

No error

### Reproduction

My webkpack config : 

```const webpack = require('webpack');
module.exports = {
	css : {
		loaderOptions : {
			sass : {
				prependData : `@import "@/assets/style/variables.scss";
            `
			}
		}
	},
	pluginOptions : {
		i18n : {
			locale         : 'en',
			fallbackLocale : 'en',
			localeDir      : 'locales',
			enableInSFC    : true
		}
	},
	chainWebpack : config => {
		config.module
			.rule('i18n')
			.resourceQuery(/blockType=i18n/)
			.type('javascript/auto')
			.use('i18n', new webpack.DefinePlugin({
				__INTLIFY_PROD_DEVTOOLS__ : true
			})
			)
			.loader('@intlify/vue-i18n-loader');
	},
	publicPath : './'

};
```

### Screenshot

![image](https://user-images.githubusercontent.com/28560613/125355366-37111300-e365-11eb-9f89-2d06468a0aa7.png)

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/vue-i18n-next/discussions)

**Comment by kazupon** [comment]
Thank you for feedback!
I've just fixed at v9.1.7
https://github.com/intlify/vue-i18n-next/releases/tag/v9.1.7

please check it!

**Comment by kazupon** [comment]
close, due to so it's had passed 5 days.
if you have anything issue, you can open a new issue.

Thanks.
